mesonbuild/interpreter/interpreterobjects.py

```diff
@@ -645,6 +646,19 @@ def _full_path(self) -> str:
         assert path is not None
         return path
 
+    @noPosargs
+    @noKwargs
+    @FeatureNew('ExternalProgram.cmd_array', '1.6.1')
+    def cmd_array_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> T.List[str]:
+        return self._cmd_array()
+
+    def _cmd_array(self) -> T.List[str]:
```

Comment:
We don't need a proxy function here since it is only used once. The reason `self._full_path()` exists is because multiple public methods invoke it.